nupic/frameworks/opf/clamodel.py

```diff
@@ -441,6 +443,21 @@ def _getSensorInputRecord(self, inputRecord):
                        sequenceReset=resetOut,
                        category=inputRecordCategory)
 
+  def _getClassifierInputRecord(self, inputRecord):
```

Comment:
Can we put this logic in `_classificationCompute`, and have it store this information in the `inference` variable that it returns? That way we don't have to access `self._classifierInputEncoder` like this.